src/style.css

```diff
@@ -1,3 +1,78 @@
+
+@media (max-width: 1024px) {
+  .header {
+    max-width: 1024px;
+  }
+}
```

Comment:
There is no need to limit the max-width, so page looks okay on large screens too.